VAMobile/src/components/FormWrapper/FormFields/VATextInput.test.tsx

```diff
@@ -1,14 +1,14 @@
 import 'react-native'
 import React from 'react'
-import { Pressable, TextInput } from 'react-native'
+import { TextInput } from 'react-native'
 // Note: test renderer must be required after react-native.
 import 'jest-styled-components'
-import { ReactTestInstance, act } from 'react-test-renderer'
+import { ReactTestInstance } from 'react-test-renderer'
+import { screen } from '@testing-library/react-native'
```

Comment:
this import should be grouped with the top level imports and not the bottom

Reply:
Updated. I'm actually getting the `screen` from the `testUtils` file now, along with other requirements.